File: find-length-of-number/c#/using-recursion.cs
using System;

public class NumberLength
{
    public static void Main(string[] args)
    {
        int num = 123456;
        
        Console.WriteLine("Length of number is " + FindLength(num));
    }
    
    public static int FindLength(int num)
    {
        if(num == 0)
        {
            return 0;
        }
    
        return 1 + FindLength(num / 10);
    }
}

File: find-length-of-number/c-c++/using-recursion.c
#include <stdio.h>

int find_length(int num) 
{
    if(num == 0) {
        return 0;
    }
    
    return 1 + find_length(num/10);
}

int main() {
    int num = 123456;

    printf("%d", find_length(num));

    return 0;
}